packages/cli/generation/ir-generator/src/converters/services/convertHttpService.ts

```diff
@@ -30,6 +30,7 @@ export function convertHttpService({
             fernFilepath,
         },
         basePath: serviceDefinition["base-path"],
```

Comment:
Isn't this effectively a break anyway? Since this will be `basePath: /{appId}/` which will break old generators. All to say do we even need to v2?

Reply:
if we don't v2 then we guarantee that any runs of `fern generate` with non upgraded generators will fail as well (which will likely be the common case)

Reply:
by keeping around the old field we wont be breaking usage of non-upgraded generators

Reply:
Makes sense- and if you want to use the new path params thing you have to make sure the generators you’re using support the v2?

Reply:
exactly